src/americana.js

```diff
@@ -28,15 +28,20 @@ upgradeLegacyHash();
 
 loadRTLPlugin();
 
-export const map = createMap(window, (shields) => shieldDefLoad(shields), {
-  container: "map", // container id
-  hash: "map",
-  antialias: true,
-  style: buildStyle(),
-  center: [-94, 40.5],
-  zoom: 4,
-  attributionControl: false,
-});
+const initializeMap = async () => {
+  await document.fonts?.ready;
```

Comment:
I don’t think the map should be blocked on loading the Web font, just as it shouldn’t be blocked on loading the fontstack or sprites. If something causes the Web font to fail to load, that isn’t such a critical problem that the user should be left with a blank page. This is already how WebKit handles Web fonts for native text, thanks to Apple’s perfectionism, and it causes all sorts of problems like https://github.com/ZeLonewolf/openstreetmap-americana/pull/955#issuecomment-1751888674.

Either this behavior should be limited to headless mode, or we should continue to load the map right away but purge the shield-related sprites as soon as the Web font loads. I’d prefer the latter because it would address the issue where some Safari users are seeing blank shields.

Reply:
While it's not possible to make styleimagemissing async, purging the shields when the font loads should work.

And probably will avoid any visual glitches.